modules/proc-providing-feedback-on-redhat-documentation.adoc

```diff
@@ -0,0 +1,46 @@
+[preface]
+[id="proc-providing-feedback-on-redhat-documentation"]
+= Providing feedback on Red Hat documentation
+
+We appreciate your feedback on our documentation.
+To provide feedback, you can highlight the text in a document and add comments.
+
+This section explains how to submit feedback.
+
+.Prerequisites
+
+* You are logged in to the Red Hat Customer Portal.
+* In the Red Hat Customer Portal, view the document in *Multi-page HTML* format.
+
+.Procedure
+
+To provide your feedback, perform the following steps:
```

Comment:
```suggestion

```

Reply:
Superfluous sentence. Either use `.Procedure` to signal the start of a task or an introductory sentence, not both.

Now that I'm thinking about it, I would prefer to get rid of the `.Prerequisites` and `.Procedure` titles and just keep this sentence rephrased as:

"To provide us with your feedback, do the following:"